src/base/searchengine.h

```diff
@@ -38,10 +40,119 @@
 class QProcess;
 class QTimer;
 
+template <class T, std::size_t N>
+class Version
```

Comment:
I wish this worked in the usual way: it should parse version string with/without trailing zeros and it should print version in following format: min items count + trailing items unless the last item is zero.